frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-details-page/feature-flag-details-page-content/feature-flag-details-page-content.component.ts

```diff
@@ -6,11 +6,12 @@ import { FeatureFlagExclusionsSectionCardComponent } from './feature-flag-exclus
 import { FeatureFlagExposuresSectionCardComponent } from './feature-flag-exposures-section-card/feature-flag-exposures-section-card.component';
 import { FeatureFlagOverviewDetailsSectionCardComponent } from './feature-flag-overview-details-section-card/feature-flag-overview-details-section-card.component';
 import { FeatureFlagsService } from '../../../../../../core/feature-flags/feature-flags.service';
-import { Observable, Subscription } from 'rxjs';
+import { Observable, Subscription, combineLatest } from 'rxjs';
 import { FeatureFlag } from '../../../../../../core/feature-flags/store/feature-flags.model';
-import { ActivatedRoute } from '@angular/router';
+import { ActivatedRoute, NavigationEnd, Router } from '@angular/router';
 import { SharedModule } from '../../../../../../shared/shared.module';
```

Comment:
What's the purpose of `SharedModule`? I noticed that if I removed that from 'CommonDetailsParticipantListTableComponent' if fixed this problem, and didn't result in any other problems - but I probably didn't see them.